test/tests/concrete_moisture_heat_transfer/concrete_thermal_moisture_properties.i

```diff
@@ -0,0 +1,168 @@
+[Mesh]
+  [square]
+    type = GeneratedMeshGenerator
+    dim = 2
+  []
+[]
+
+[Problem]
+  solve = false
+[]
+
+[AuxVariables]
+  [temperature]
+    order = CONSTANT
+    family = MONOMIAL
```

Comment:
We need to provide initial condition, else I am getting mismatched value for the material properties at the initial time step.

`initial_condition = 20`